README.md

```diff
@@ -182,7 +182,7 @@ https://jwst-pipeline.readthedocs.io/en/latest/
 
 To build the docs yourself, clone this repository and build the documentation with:
 
-    pip install -e ".[docs]"
+    pip install -e ".[docs]**
```

Comment:
Just need to fix this one little item and it should be ready to merge.

Reply:
That would be stray fingers wondering about the keyboard. Will fix.